web/src/lib/components/forms/edit-user-form.svelte

```diff
@@ -68,8 +69,12 @@
 </script>
 
 <div
-  class="max-h-screen w-[500px] max-w-[95vw] overflow-y-auto rounded-3xl border bg-immich-bg p-4 py-8 shadow-sm dark:border-immich-dark-gray dark:bg-immich-dark-gray dark:text-immich-dark-fg"
+  class="relative max-h-screen w-[500px] max-w-[95vw] overflow-y-auto rounded-3xl border bg-immich-bg p-4 py-8 shadow-sm dark:border-immich-dark-gray dark:bg-immich-dark-gray dark:text-immich-dark-fg"
 >
+  <div class="absolute inset-y-0 right-0 px-2 py-2 h-fit">
+    <CircleIconButton icon={mdiClose} on:click={() => dispatch('close')} />
```

Comment:
While you are here, do you mind adding types for the dispatcher? In general they are missing in many places and I think it would be easy enough to add whenever we update a file that is missing them.

Reply:
Of course